cmd/test_utils/tx.go

```diff
@@ -63,7 +63,7 @@ func GenTxWithMsg(messages []sdk.Msg) (auth.StdTx, error) {
 	if err != nil {
 		return auth.StdTx{}, err
 	}
-	stdSignMsg.Fee.Gas = 400000
+	stdSignMsg.Fee.Gas = 10000000
```

Comment:
Hmm, I was thinking of removing this Gas value at a stage as we don't use gas system and this gas system is introduced by cosmos sdk. Not as part of this PR but I would like to remove this check soon.

Reply:
https://github.com/Pylons-tech/pylons_sdk/issues/61 is for this comment.